gapis/api/vulkan/templates/vk_spy_helpers.cpp.tmpl

```diff
@@ -254,6 +256,37 @@ uint32_t VulkanSpy::SpyOverride_vkCreateInstance(VkInstanceCreateInfo *pCreateIn
     // Actually call vkCreateInstance, and keep track of the result.
     uint32_t result = create_instance(pCreateInfo, pAllocator, pInstance);
 
+    // Send a header with Vulkan info added if we haven't done so.
```

Comment:
Not really introduced in this CL, but I'm not keen on this much bespoke code in a template.
Can it not be moved to a regular C++ file?

Reply:
I agree, having such a lot of c++ code in template file is quite weird. I can work on that in another CL